### PR TITLE
Information about vesting

### PR DESCRIPTION
**Pull request by AndreyKortosov**
Information about vesting in the Blockchain and Polkaswap

### CODE REVIEW

File: src/vesting-tokens-polkaswap.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - - meta
+    - name: keywords
+      content: SORA ecosystem, vesting tokens, distribute in period, gradual distribution, Business Cases for Vesting
+---
+
+# Vesting in Cryptocurrency
+
+Vesting is a method used in cryptocurrency projects to gradually release tokens over a specific period rather than distributing them all at once. Vesting is commonly applied to tokens allocated to project teams, early investors, and contributors, with the goal of encouraging long-term commitment and reducing the risk of sudden large sell-offs that could destabilize the token’s value.
+
+For more details, see[Vesting tokens in SORA](/vesting-tokens.md)
```
Comment (ZlayaMorda) [inline src/vesting-tokens-polkaswap.md]: doesn't work, / - means root, there maybe should be without /

File: src/vesting-tokens-polkaswap.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Vesting in Polkaswap
+
+In Polkaswap, you can enable token vesting when sending tokens. Open the _Send Token_ screen and switch on the _Enable Token Vesting_ toggle. Then, fill in the details for the vesting transaction:
+
+- **Unlock Frequency**: Set how frequently tokens will unlock in the recipient's wallet. Options include 1 day, 7 days, 30 days, 60 days, and 90 days.
+- **Vesting Percentage**: Specify the percentage of tokens to unlock in each interval.
+
+![](.gitbook/assets/vesting-polkaswap-send.png)
+
+Below this section, you’ll see the _Network Fee_, which varies based on the total amount of tokens to be vested.
```
Comment (ZlayaMorda) [inline src/vesting-tokens-polkaswap.md]: based on the percent (count of periods)

File: src/vesting-tokens.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+### LinearVestingSchedule
+
+Using the _vestedTransfer_ extrinsic with _LinearVestingSchedule_ enables the distribution of tokens evenly over a specified period and amount.
+
+![](.gitbook/assets/vesting-linear.png)
+
+Parameters:
+
+- **assetId**: The code of the token.
+- **start**: The current block number on the SORA blockchain.
```
Comment (ZlayaMorda) [inline src/vesting-tokens.md]: from front current, but may be set to any

File: src/vesting-tokens-polkaswap.md (hunk 1 above)
Comment (AndreyKortosov) [reply src/vesting-tokens-polkaswap.md]: Fixed it

File: src/vesting-tokens-polkaswap.md (hunk 2 above)
Comment (AndreyKortosov) [reply src/vesting-tokens-polkaswap.md]: Fixed it

File: src/vesting-tokens.md (hunk 3 above)
Comment (AndreyKortosov) [reply src/vesting-tokens.md]: Fixed it

File: src/vesting-tokens.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+### LinearVestingSchedule
+
+Using the _vestedTransfer_ extrinsic with _LinearVestingSchedule_ enables the distribution of tokens evenly over a specified period and amount.
+
+![](.gitbook/assets/vesting-linear.png)
+
+Parameters:
+
+- **assetId**: The code of the token.
```
Comment (WRRicht3r) [inline src/vesting-tokens.md]:
```suggestion
- **assetId**: The token address.
```

File: src/vesting-tokens.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+- **start**: The number of block in the SORA blockchain.
```
Comment (WRRicht3r) [inline src/vesting-tokens.md]:
```suggestion
- **start**: The block number in the SORA blockchain.
```

File: src/vesting-tokens.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+### LinearPendingVestingSchedule
+
+The _vestedTransfer_ extrinsic with _LinearPendingVestingSchedule_ allows for delayed token distribution over a specified period and amount, managed by an external initiator.
+
+![](.gitbook/assets/vesting-linear-pending.png)
+
+Parameters:
+
+- **assetId**: The code of the token.
```
Comment (WRRicht3r) [inline src/vesting-tokens.md]:
```suggestion
- **assetId**: The token address.
```

File: src/vesting-tokens.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+- **managerId**: The ID of the entity that will initiate vesting via the unlock_pending_schedule_by_manager() extrinsic.
+- **period**: Unlock intervals for tokens in recipient wallets, measured in blocks.
+- **periodCount**: The number of periods, calculated as a percentage of the total amount.
+- **perPeriod**: The amount of tokens to distribute per period, calculated as the initial amount multiplied by the vesting percentage.
+- **remainderAmount**: If any tokens remain after calculating the per-period distribution, they can be added here to be included in the final distribution.
+
+## Vesting tokens in Polkaswap
+
+You can also access vesting functionality through Polkaswap. Simply open the "Send Token" feature and toggle on _Enable Token Vesting_. More details are available on the [Vesting tokens in Polkaswap](vesting-tokens-polkaswap.md) page.
```
Comment (WRRicht3r) [inline src/vesting-tokens.md]:
```suggestion
You can also access vesting functionality through Polkaswap. Open the "Send Token" feature and toggle _Enable Token Vesting_. More details are available on the [Vesting tokens in Polkaswap](vesting-tokens-polkaswap.md) page.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**